operator/controllers/kyma_controller.go

```diff
@@ -317,32 +342,38 @@ func (r *KymaReconciler) SyncConditionsWithModuleStates(ctx context.Context, kym
 	return statusUpdateRequired, err
 }
 
-func (r *KymaReconciler) CreateOrUpdateModules(ctx context.Context, kyma *operatorv1alpha1.Kyma, modules util.Modules) (bool, error) {
+func (r *KymaReconciler) CreateOrUpdateModules(ctx context.Context, kyma *operatorv1alpha1.Kyma,
+	modules util.Modules,
+) (bool, error) {
 	logger := log.FromContext(ctx).WithName(client.ObjectKey{Name: kyma.Name, Namespace: kyma.Namespace}.String())
 	kymaSyncNecessary := false
+
 	for name, module := range modules {
 		// either the template in the condition is outdated (reflected by a generation change on the template)
-		//or the template that is supposed to be applied changed (e.g. because the kyma spec changed)
```

Comment:
please leave this comment in, I tend to forget it myself quite often :')

Reply:
Yes, bring back already, I assume it's accidentally removed, initial idea is resolve the complain that there must be space between comment symbol and actual text